### ISSUE TITLE
r2pm doesn't works if Cmake's default generator isn't make

### ISSUE THREAD

**Issue by GiuseppeCesarano** [radareorg/r2pm#73]
**Describe the issue**  
r2pm doesn't build packages if the cmake's default generator isn't make.

**How to reproduce?**
Simply setting cmake's default generator to anything else than makefiles. Eg: Having CMAKE_GENERATOR=Ninja 

**Expected behavior**
Still building the file using the default generator.

**Possible fix**
Using `cmake --build .` instead of `make` or forcing cmake to use make

**Comment by qbarrand** [comment]
Hi @GiuseppeCesarano. I must confess that I'm a bit puzzled here - r2pm does not call `cmake` itself, it merely executes the commands from a manifest. Can you explain precisely how you hit that problem?

**Comment by GiuseppeCesarano** [comment]
Yes. I was trying to install **r2ghidra-dec** with the suggested command `r2pm -i r2ghidra-dec`. Somewhere, CMake is called, and build files are generated, than later make is called too on the generated files, but i had the default CMake generator to Ninja so the make call failed.
